### ISSUE TITLE
Sentry 4.0 incompatibility with `ActiveSupport::Notifications.publish`

### ISSUE THREAD

**Issue by agrobbin** [getsentry/sentry-ruby#1131]
**Describe the bug**

When testing our upgrade to Sentry 4.0, we ran into an issue with another dependency calling `ActiveSupport::Notifications.publish` and Sentry's `ActiveSupportLogger` trying to call `to_i` on a non-`Time` object. In this case, it is Active Admin's [`load!` method](https://github.com/activeadmin/activeadmin/blob/v2.8.1/lib/active_admin/application.rb#L110-L120), though I'd imagine any use of `ActiveSupport::Notifications.publish` will cause a similar issue.

**To Reproduce**

Call `ActiveSupport::Notifications.publish`.

**Expected behavior**

`ActiveSupportLogger` should not raise an exception.

**Actual behavior**

`ActiveSupportLogger` raises an `undefined method 'to_i'` exception.

**Environment**
 - Ruby Version: 2.7.2
 - SDK Version: 4.0.0
 - Integration Versions (if any):
  - Rails 6.0.3.4
  - Active Admin 2.8.1

**Raven Config**

This is not necessary but could be helpful.

```ruby
Sentry.init do |config|
  # Send Sentry notifications asynchronously by default
  config.async = ->(event) { SentryJob.perform_later(event) }

  config.breadcrumbs_logger = [:active_support_logger]
end
```

**Comment by st0012** [comment]
@agrobbin thanks for reporting, should be fixed by https://github.com/getsentry/sentry-ruby/pull/1132

**Comment by agrobbin** [comment]
Wow, that was fast @st0012, thanks!! Looking forward to being able to try this out once it's released.

**Comment by st0012** [comment]
@agrobbin you can still try it without setting `config.breadcrumbs_logger = [:active_support_logger]`. I guess you didn't have that when using the old SDK?

**Comment by st0012** [comment]
ah fat finger clicked reopen 🤦‍♂️ closing again

**Comment by agrobbin** [comment]
@st0012 thanks!

**Comment by st0012** [comment]
fyi the fix has been released in `4.0.1`
